docs/SetupDatabaseForAKS.md

```diff
@@ -0,0 +1,108 @@
+
+### Scale Performance
+
+Cosmos DB performance can be adjusted using the following parameters.
+
+#### Request Units (RU)
+
+Request unit is a performance currency abstracting the system resources such as CPU, IOPS, and memory that are required to perform the database operations supported by Azure Cosmos DB.
+You can scale the throughput of Cosmos DB by specifying `cosmosBaseResourceUnit` (which applies to all the tables) in [schema-loading-custom-values](https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/schema-loading-custom-values.yaml).
+Scalar schema tool abstracts Request Unit of Cosmos DB with RU. So please set the appropriate value.
+
+#### Autoscale
+
+Autoscale provisioned throughput in Azure Cosmos DB allows you to scale the throughput (RU/s) of your database or container automatically and instantly.
+The throughput is scaled based on the usage, without impacting the availability, latency, throughput, or performance of the workload.
+By default, the scalardl schema tool enables autoscale of RU for all tables: RU is scaled in or out between 10% and 100% of a specified RU depending on a workload.
+
+### Monitor Cosmos DB
+
+By default, monitoring is enabled on Cosmos DB.
+
+More information can be found in [Monitor Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/monitor-cosmos-db) documentation.
+
+## Azure Database for MySQL
+
+Azure Database for MySQL is a fully managed MySQL database provided by Azure. You can create a MySQL database by selecting a database instance and storage according to your configurations. The IOPS is calculated based on the compute instance and storage you choose.
+In this section, you will create an Azure Database for MySQL.
+
+### Recommendations
+
+* You should choose Flexible or Single server deployment for Azure Database for MySQL. You can check the [official documentation](https://docs.microsoft.com/en-us/azure/mysql/select-right-deployment-type) to learn more.
+* You should select a deployment that suits your IOPS requirements.
+    * For Single server deployments, IOPS is defined on the amount of storage provisioned(3 IOPS/GB).
+    * For Flexible server deployments, additional IOPS can be provisioned and the maximum threshold will be defined by the storage and the database instance chosen.
+
+### Steps
+
+For creating an Azure Database for MySQL, 
+* For Single Server deployment, follow this [official documentation](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal).
+* For creating a flexible server deployment, follow this [official documentation](https://docs.microsoft.com/en-us/azure/mysql/flexible-server/quickstart-create-server-portal).
+
+### Scale Performance
+
+**Scaling storage**:- You can enable the `storage auto grow` feature to automatically increase storage when it reaches a threshold without impacting the workload.
+
+You can scale resources based on the deployment you choose.
```

Comment:
I think we need to reduce the use of `you can`.

Reply:
I have reduced the use of the phrase `you can`.